src/Tasks/Microsoft.NETFramework.CurrentVersion.targets

```diff
@@ -93,6 +93,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <PropertyGroup>
       <!-- Does one of our dependencies reference a System.Runtime-based portable library? -->
       <_HasReferenceToSystemRuntime Condition="'$(DependsOnSystemRuntime)' == 'true'">true</_HasReferenceToSystemRuntime>
+      <_HasReferenceToSystemRuntime Condition="'$(DependsOnNETStandard)' == 'true'">true</_HasReferenceToSystemRuntime>
```

Comment:
Do we need all of the facades if there is a reference to netstandard.dll but no reference to system.runtime? Seems like there could be a perf win if only netstandard.dll is in the closure if we only pass along the netstandard.dll facade.

Reply:
If it turns out we can't do that, can we at least rename this to _NeedsFacades or something? We shouldn't have _HasReferenceToSystemRuntime=true with no reference to System.Runtime.

Reply:
/cc @weshaggard @alexghi
I believe we don't need everything if only netstandard.dll is referenced.  

This would only be true for the targeting packs: in those cases the facades referencing non-facades.  We couldn't make that assumption in the Extensions targets in the SDK since netstandard referenced other assemblies that were injected (eg: netstandard.dll > System.ValueTuple, netstandard.dll > System.Net.Http, etc).